wiki/Submission/en.md

```diff
@@ -1,150 +1,31 @@
-
-You can also select the following via checkboxes under the message input box:
-
--   **Receive email notification on reply:** When this box is checked, once someone replies to your beatmap's thread, you will receive a notification e-mails in the e-mail address you provided when you registered in osu!. You will receive no further notifications for any more replies until you visit the thread.
--   **Load in browser after submission:** If this box is checked, after the upload is complete and you click "Submit", your default browser will attempt to display the page of your beatmap's forum thread.
-
-### The Beatmap Modding Process
-
-*Main article: [Modding](/wiki/Modding).*
-
-After you submit your beatmap, it will be available for downloading by the osu! forum members and moderators, who may provide comments and feedback on it (commonly referred to as "modding" a beatmap). In [Extended Chat](/wiki/Chat_Console), the status of someone who has opened another person's beatmap in the Editor is set as "Modding" and their panel turns green. Everyone is allowed - *encouraged*, in fact - to mod beatmaps, but only the modding of moderators determines how close a beatmap is to be Ranked.
-
-Moderators can give a symbol to the beatmap's thread based on their evaluation of the beatmap. This symbol is displayed next to its subject in the subforum and next to the title of its first post. The possible symbols a beatmap can have are (from best to worst):
-
--   ![Heart icon](/wiki/shared/icon/heart.gif "Heart icon") Heart / ![Fire icon](/wiki/shared/icon/flame.gif "Fire icon") Flame
--   ![Thinking icon](/wiki/shared/icon/bubble.gif "Thinking icon") Thought bubble (Bubble for short)
--   ![Star icon](/wiki/shared/icon/star.gif "Star icon") Star
--   No symbol
--   ![Heart-broken icon](/wiki/shared/icon/broken-heart.gif "Heart-broken icon") Heart-broken
--   ![Nuke icon](/wiki/shared/icon/nuke.gif "Nuke icon") Nuke
-    -   *Graveyard* if inactive for a month.
-
-#### ![Heart icon](/wiki/shared/icon/heart.gif "Heart icon") Heart
-
-Once a second Beatmap Nominator nominates a beatmap with a ![Thinking icon](/wiki/shared/icon/bubble.gif "Thinking icon") Thought Bubble symbol, he/she qualifies it and it is automatically given this symbol when moved to the Ranked/Approved Beatmaps forum.
+`Upload Beatmap...` from the `File` dropdown in the editor (shortcut: `Ctrl` + `Shift` + `U`) will open the "Beatmap Submission System" (BSS) window. This initially displays lists resources for users to ensure their maps are suitable for ranking, ask mapping-related questions, and find feedback for their beatmaps.
```

Comment:
```suggestion
Selecting `Upload Beatmap...` from the `File` dropdown in the editor (shortcut: `Ctrl` + `Shift` + `U`) will open the **Beatmap Submission System** (***BSS***) window. This initially lists resources to help users ask mapping-related questions, find feedback for their beatmaps, and ensure their maps are suitable for ranking.
```
parallelism isn't too great here but u can get away with it if u put the irregular phrase last 💭 
and bolded the main topic of article